components/item.tsx

```
@@ -83,39 +95,34 @@ export default function Item(props: ItemProps) {
-												? typeof props.category[0] === 'string'
-													? props.category[0]
-													: props.category[0]?.name
-												: typeof props.category === 'string'
-													? props.category
-													: props.category?.name
-										}
-										variant="star"
-										size="sm"
-										className="opacity-0 group-hover:opacity-100 transition-opacity duration-300"
-									/>
+									<div onClick={(e) => e.stopPropagation()}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)